examples/spectrum-scanner/main.c

```diff
@@ -0,0 +1,145 @@
+/*
+ * Copyright (C) 2017 Eistec AB
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup    app_spectrum_scanner
+ * @brief       Scanner application to find free channels
+ * @{
+ *
+ * @file
+ * @brief       Spectrum scanner application for RIOT
+ *
+ * @author      Joakim Nohlgård <joakim.nohlgard@eistec.se>
+ *
+ * @}
+ */
+
+#include <stdint.h>
```

Comment:
Oh, it is an actual bug that the PIC32 (and the RISC-V) toolchain found an actual bug.

```suggestion
#include <stdint.h>
#include <stdio.h>
```